src/wp-admin/user-edit.php

```diff
@@ -241,213 +255,225 @@
 <h2 class="user-profile-personal-options"><?php _e( 'Personal Options' ); ?></h2>
 
 <table class="form-table">
-<?php if ( ! ( IS_PROFILE_PAGE && ! $user_can_edit ) ) : ?>
+		<?php if ( ! ( IS_PROFILE_PAGE && ! $user_can_edit ) ) : ?>
 	<tr class="user-rich-editing-wrap">
 		<th scope="row"><?php _e( 'Visual Editor' ); ?></th>
-		<td><label for="rich_editing"><input name="rich_editing" type="checkbox" id="rich_editing" value="false" <?php if ( ! empty( $profileuser->rich_editing ) ) checked( 'false', $profileuser->rich_editing ); ?> /> <?php _e( 'Disable the visual editor when writing' ); ?></label></td>
+		<td><label for="rich_editing"><input name="rich_editing" type="checkbox" id="rich_editing" value="false" <?php checked( 'false', $profileuser->rich_editing ); ?> /> <?php _e( 'Disable the visual editor when writing' ); ?></label></td>
```

Comment:
The check must have been deemed unnecessary since the value is always set to false in HTML.

![Screenshot 2022-07-23 at 17 42 40](https://user-images.githubusercontent.com/7713923/180609944-5a352c42-bf8f-45b1-9b36-01e250b91b6b.png)

Reply:
This is one of those bad choices for UI. Most understandable is to have the value "on" when the option is chosen. Disabling is backwards, and makes the logic hard to follow, especially when it's a checkbox (which doesn't send in the form unless checked).
I would be tempted to flip it to "Enable visual editor" and default to true, but it would be a breaking change.

Reply:
Agreed.

Reply:
Resolving this since the previous behavior of the code is still intact.